python/strings_udf/strings_udf/__init__.py

```diff
@@ -87,6 +88,26 @@ def _get_ptx_file():
         return regular_result[1]
 
 
+default_heap_size = int(2e6)
```

Comment:
I feel like this needs a big comment on why this is default. This number seems a bit high considering 10M rows of 100 bytes each would only be 1GB. I suppose it  makes sense since it is the largest strings column in bytes possible.

Would it be too strange to make this configurable through an environment variable perhaps?
At least for this release until we can come up with a good metric for setting it.

Reply:
I introduced an environment variable to control this. I'm still open to discussion here as to what to set the value as. Indeed I did base it on the size of a strings column.